sklearn/metrics/pairwise.py

```diff
@@ -977,7 +977,8 @@ def paired_distances(X, Y, *, metric="euclidean", **kwds):
 
 
 # Kernels
-def linear_kernel(X, Y=None, dense_output=True):
+@_deprecate_positional_args
+def linear_kernel(X, Y=None, *, dense_output=True):
```

Comment:
I _think_ in the PR I touched this file, we decided not to touch the kernels, but I'm happy with this.

Reply:
https://github.com/scikit-learn/scikit-learn/pull/16982#discussion_r413763579